tests/internal/e2e/run_e2e_tests.sh

```diff
@@ -25,6 +83,7 @@ if [ "$GITHUB_ACTIONS" = "true" ] || [ "$TESTING_IN_CONTAINER" = "True" ] ; then
         echo "Starting xvfb..."
         Xvfb :99 -screen 0 1920x1080x24 &
     fi
+    set_specs_params
```

Comment:
should call this on line 80 before entering the branch, not to duplicate

Reply:
Refactored to only call `set_specs_params` once in 26119a9b2016bce045ee2019c3eac106dafcea3b.